download-utils/src/lib.rs

```diff
@@ -1,238 +1,22 @@
 #![allow(clippy::arithmetic_side_effects)]
+pub use solana_file_downloader::DownloadProgressRecord;
 use {
-    console::Emoji,
-    indicatif::{ProgressBar, ProgressStyle},
     log::*,
+    solana_file_downloader::{download_file, DownloadProgressCallbackOption},
```

Comment:
Can you merge the `solana_file_downloader` imports here?

Reply:
I want to re-export DownloadProgressRecord specifically because it's part of the public API via download_snapshot_archive. If it's not re-exported then `solana_validator` needs to add `solana_file_downloader` as a direct dependency to use `DownloadProgressRecord` in `download_snapshot_archive`